packages/deploy/src/utils/deploy.ts

```diff
@@ -0,0 +1,29 @@
+import { Artifact, ContractArtifact, DeployContractRequest, RequestArtifact } from '../models';
+
+export function reduceArtifactSize(req: RequestArtifact): Artifact | undefined {
```

Comment:
Maybe there is a better name? Initially, I thought it would be a compression of some sort.
But it's an extraction.

Reply:
Yup, that name made sense for the the first implementation, then I refactored and now looks weird. Thanks!